src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -197,48 +206,70 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
     }
 
     /**
-     * Sucess callback for classification change
+     * File classification fetch success callback.
      *
-     * @private
+     * @param {ClassificationInfo} classification - Info about the file's classification
      * @return {void}
      */
-    classifiationChangeSuccessCallback = (file: BoxItem) => {
+    fetchClassificationSuccessCallback = (classification: ClassificationInfo): void => {
         this.setState({
-            file,
-            isLoading: false,
+            classification,
+            classificationError: undefined,
+            isLoadingClassification: false,
         });
     };
 
     /**
-     * Error callback for classification change
+     * Handles a failed file classification fetch
      *
      * @private
+     * @param {ElementsXhrError} error - API error
+     * @param {string} code - Error code
      * @return {void}
      */
-    classifiationChangeErrorCallback = () => {
+    fetchClassificationErrorCallback = (error: ElementsXhrError, code: string): void => {
+        let classificationError;
+
+        if (getProp(error, 'status') === HTTP_STATUS_CODE_FORBIDDEN) {
```

Comment:
Do we want to show an error message in this case? I am not sure, but I think it should silently fail

Reply:
Priyajeet also brought this up. I'll take it out, since external devs will see in a network req that the error is a 403